pkilint/etsi/en_319_412_5.py

```diff
@@ -332,17 +352,25 @@ def retrieve_qualified_statement_id(cls, node):
     def __init__(self, certificate_type: etsi_constants.CertificateType):
         allowances = {}
 
-        if certificate_type in etsi_constants.EU_QWAC_TYPES:
+        if certificate_type in etsi_constants.EU:
+            # Table 2: 4.2.1
             allowances[en_319_412_5.id_etsi_qcs_QcCompliance] = Rfc2119Word.MUST
+            # Table 2: 4.2.4
             allowances[en_319_412_5.id_etsi_qcs_QcCClegislation] = Rfc2119Word.MUST_NOT
-            allowances[en_319_412_5.id_etsi_qcs_QcType] = Rfc2119Word.MUST
+            # Table 2: 4.2.2
+            if certificate_type in etsi_constants.EU_SSCD:
+                allowances[en_319_412_5.id_etsi_qcs_QcSSCD] = Rfc2119Word.MUST
+
+            # Table 2: 4.2.3 (QWAC is Annex IV)
+            if certificate_type in etsi_constants.EU_QWAC_TYPES:
+                allowances[en_319_412_5.id_etsi_qcs_QcType] = Rfc2119Word.MUST
+                # PR Question: Table 2, 4.2.2 only defines MUST, is the MUST_NOT also from 412-5 somewhere?
+                allowances[en_319_412_5.id_etsi_qcs_QcSSCD] = Rfc2119Word.MUST_NOT
```

Comment:
In the referenced 412-5 / 5 table I couldn't find the `MUST_NOT` case anywhere, did I overlook something?

Reply:
My take is that eIDAS defines a QCSD in the context of e-signatures and e-seals https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32014R0910#:~:text=for%20automated%20processing.-,ANNEX%20II,-REQUIREMENTS%20FOR%20QUALIFIED so it's not relevant to a QWAC

Reply:
Thanks, that's clear!